index.html

```diff
@@ -543,6 +543,111 @@ <h6>JWT Decoding</h6>
+        reduce production rules to a single, simple, set of instructions that
+        any implementer can easily meet if they possess a software library
+        supporting [[RFC7515]] or [[RFC7519]].
+      </p>
+
+      <section>
+        <h3>In addition to...</h3>
+
+        <p>
+          There are several members (claims) of the
+          <code>Credential</code> which will need to be translated to their JOSE
+          form, and included next to the <code>vc</code> or
+          <code>vp</code> member in the <code>payload</code>.
+        </p>
+
+        <p>
+          If a member is not present in the <code>Credential</code> it MUST NOT
+          be present in the <code>VerifiableCredential</code> as either a claim
+          in the payload or a claim in the <code>vc</code> attribute of the
+          payload.
+        </p>
+
+        <p>
+          We start with an empty header, and payload objects, and we add members
+          to the header and the payload based on the content in the
+          <code>Credential</code>.
+        </p>
+
+        <section>
+          <h4>issuer -> iss</h4>
+
+          <p>
+            This member MUST be present in the
+            <code>payload.vc.issuer</code> attribute as either a string or an
+            <code>object</code> with and <code>id</code>.
+          </p>
+
+          <p>This member MUST be present <code>payload.iss</code>.</p>
+
+          <p>
+            In the case that <code>payload.vc.issuer</code> is an object,
+            <code>payload.iss</code> MUST be <code>payload.vc.issuer.id</code>.
+          </p>
+
+          <p>
+            In the case that
+            <code>payload.vc.issuer</code> is a string,
+            <code>payload.iss</code> must be
+            <code>payload.vc.issuer</code>
+          </p>
+        </section>
+
+        <section>
+          <h4>issuanceDate -> nbf</h4>
+
+          <p>
+            This member MUST be present in the
+            <code>payload.vc.issuanceDate</code> attribute as an XMLDateTime
+            String.
+          </p>
+
+          <p>
+            This member MUST be present <code>payload.nbf</code> as a unix
+            timestamp.
+          </p>
+
+          <p>
+            In the case that the <code>issuanceDate</code> includes leap
+            seconds, it is not possible to detect them when the date time is
+            represented in <code>nbf</code>
+          </p>
+        </section>
+
+        <p class="issue">This section needs to be defined.</p>
+
+        <section>
+          <h4>issuance</h4>
+
+          <p>
+            The header and payload converted into a JWT, in accorance with the
+            RFC:
+            <a href="https://www.rfc-editor.org/rfc/rfc7519#section-7.1"
+              >RFC7519 Section 7.1</a
+            >
+          </p>
+        </section>
+      </section>
+
+      <section>
+        <h3>Instead of...</h3>
+        <p class="issue">This section needs to be defined.</p>
+      </section>
```

Comment:
I don't understand what the "Instead of" section is trying to convey.

Reply:
There are 2 paths that are allowed, in v1.1:

1.  instead of...
2. in addition to...

This pull request only attempts to define `in addition to...` it's my opinion that `instead of ` should be defined as non conformant in v2.

See this thread: https://lists.w3.org/Archives/Public/public-credentials/2022Sep/0215.html

Regardless or the WG decision on this point, it's not my intention to define both paths in this PR.

Reply:
+1, deprecate `instead of`, retain `in addition to`

Reply:
I can live with either but not both!. So we should decide in this standard which we are going to employ and mandate that, regardless of what the VC DM v2 spec says.

Reply:
IMO, `instead of` is the one that should be removed... it creates a lot of implementation complexity, and it potentially destroys the usability of the data model, but thats not what this PR is about.

Reply:
It's a more natural JWT representation to use exclusively JWT claims to represent information for which there is a defined JWT representation.  As such, `instead of` seems more natural.

Reply:
Lets continue the discussion on https://github.com/w3c/vc-jwt/issues/8